tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,
+	mockHost: ServerHost = {
+		args: [],
+		newLine: '\n',
+		useCaseSensitiveFileNames: true,
+		write: (s) => lastWrittenToHost = s,
+		readFile: () => void 0,
+		writeFile: () => void 0,
+		resolvePath: () => void 0,
+		fileExists: () => false,
+		directoryExists: () => false,
+		createDirectory: () => void 0,
+		getExecutingFilePath: () => void 0,
+		getCurrentDirectory: () => void 0,
+		readDirectory: () => void 0,
+		exit: () => void 0

```

Comment:
Just use a block body for all of these `void 0` functions. 

In fact, you can write them all in method syntax.

```TypeScript
exit() {
}
```